add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: There is no error handling for invalid or missing input data.

Recommendations:
- **Input Validation**: Validate user input before processing it. Ensure that all fields are filled and that the year is a valid number.
- **Example**:
  ```php
  if (empty($title) || empty($author) || !is_numeric($year)) {
      echo "Please fill all fields correctly.";
      return;
  }
  ```